common/js/AGENTS.md

```
@@ -107,4 +111,152 @@ Access through `context.logger`:
 - Use the storage APIs for persisting data
 - Consider agent communication for complex workflows
 
-For complete documentation, visit: https://agentuity.dev/SDKs/javascript/api-reference
+## 6. How to use the documentation site, it's API's, etc.
+
+The Agentuity documentation site provides API endpoints that AI agents can use to search and read documentation programmatically.
+
+For the complete docs: https://agentuity.dev
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)